docs/en/latest/plugins/openfunction.md

```diff
@@ -0,0 +1,125 @@
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## Description
+
+The `openfunction` Plugin is used to integrate APISIX with [CNCF OpenFunction](https://openfunction.dev/) serverless platform.
+
+This Plugin can be configured on a Route and requests will be send to the configured OpenWhish API endpoint as the upstream.
+
+## Attributes
+
+| Name                        | Type    | Required | Default | Valid values | Description                                                                                                |
+| --------------------------- | ------- | -------- | ------- | ------------ | ---------------------------------------------------------------------------------------------------------- |
+| function_uri                | string  | True     |         |              | function uri. For example, `https://localhost:30858/default/function-sample`.                              |
+| ssl_verify                  | boolean | False    | true    |              | When set to `true` verifies the SSL certificate.                                                           |
+| authorization               | object  | False    |         |              | Authorization credentials to access functions of OpenFunction.                                      |
+| authorization.service_token | string  | False    |         |              | The token format is 'xx:xx' which support basic auth for ingress controller, .                                      |
+| timeout                     | integer | False    | 3000ms  | [100, ...]ms | OpenFunction action and HTTP call timeout in ms.                                                              |
+| keepalive                   | boolean | False    | true    |              | When set to `true` keeps the connection alive for reuse.                                                   |
+| keepalive_timeout           | integer | False    | 60000ms | [1000,...]ms | Time is ms for connection to remain idle without closing.                                                  |
+| keepalive_pool              | integer | False    | 5       | [1,...]      | Maximum number of requests that can be sent on this connection before closing it.                          |
+
+:::note
+
+The `timeout` attribute sets the time taken by the OpenFunction to execute, and the timeout for the HTTP client in APISIX. OpenFunction calls may take time to pull the runtime image and start the container. So, if the value is set too small, it may cause a large number of requests to fail.
+
+:::
+
+## Enabling the Plugin
+
+Before configuring the plugin, you need to have OpenFunction running. For details, please refer to [Installation](https://openfunction.dev/docs/getting-started/installation/).
+The example below shows OpenFunction installed in Helm:
+
+```shell
+#add the OpenFunction chart repository
```

Comment:
```suggestion
# add the OpenFunction chart repository
```

Reply:
Move these steps to the `Prerequisites` section (out of `Enabling the Plugin`).

Also, we need to tell users that Kubernetes is required.